javaslang/generator/Generator.scala

```diff
@@ -51,6 +51,377 @@ def generateMainClasses(): Unit = {
       val PredicateType = im.getType("java.util.function.Predicate")
       val SupplierType = im.getType("java.util.function.Supplier")
       val IteratorType = im.getType("javaslang.collection.Iterator")
+      val EitherType = im.getType("javaslang.control.Either")
+      val TryType = im.getType("javaslang.control.Try")
+      val ValidationType = im.getType("javaslang.control.Validation")
+      val FutureType = im.getType("javaslang.concurrent.Future")
+      val CheckedSupplierType = im.getType("javaslang.control.Try.CheckedSupplier")
+      val ExecutorServiceType = im.getType("java.util.concurrent.ExecutorService")
+      val ArrayType = im.getType("javaslang.collection.Array")
```

Comment:
yes, but as long as Array is here we need to include it in our changes. it is not clear yet, if we will remove it.